src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+public abstract class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    protected static int numberOfTask = 0;
+
+    public Task (String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDone() {
+        isDone = true;
+    }
+
+    public boolean isDone() {
```

Comment:
Maybe a better naming would be getDoneStatus and setDoneStatus, since both methods are actually using the same variable isDone and so perhaps naming its corresponding getter and setter in a similar format would be more standardised